frontend/src/scenes/surveys/SurveyDragHandle.tsx

```diff
@@ -17,8 +18,13 @@ export function SurveyDragHandle({
     listeners,
     isDraftSurvey,
     hasMultipleQuestions,
+    allQuestionsHaveIds,
 }: SurveyDragHandleProps): JSX.Element | null {
-    if (!isDraftSurvey || !hasMultipleQuestions) {
+    if (!hasMultipleQuestions) {
+        return null
+    }
+
+    if (!allQuestionsHaveIds && !isDraftSurvey) {
```

Comment:
`isDraftSurvey` should be removed from the conditions since now its allowed also for non draft surveys, right?

Reply:
i guess it's not really needed anymore, removed both